include/dt-bindings/pinctrl/stm32-pinctrl.h

```diff
@@ -107,4 +109,10 @@
 #define STM32_PUPDR_MASK		0x3
 #define STM32_PUPDR_SHIFT		9
 
+/* GPIO plain output value */
+#define STM32_ODR_0			(0x0<<STM32_ODR_SHIFT)
+#define STM32_ODR_1			(0x1<<STM32_ODR_SHIFT)
```

Comment:
nit: spaces around `<<`

Reply:
And checkpatch.pl didn't catch this as it is busy tracking `_TypdeDef`..